### PR TITLE
moving over tests from maestro to new repo, wasm-binder tests

### PR DESCRIPTION
**Pull request by lanseafood**
added STN, APSP tests against complex networks and edge cases from maestro

### CODE REVIEW

File: src/stn.rs [hunk 1]
```diff
@@ -658,74 +658,545 @@ mod tests {
...
+            Ok(u) => assert_eq!(
+                (4_usize, 12_usize),
+                (u.0, u.1),
+                "4 nodes, 12 edges expected to be made from given 3 edges"
+            ),
+            Err(e) => panic!("failed running stn.register_graph | {:?}", e),
+        }
+    }
+
+    //THIS TEST STILL FAILING
```
Comment (cameronwp) [inline src/stn.rs]: I'd like this test to pass. Given it doesn't appear to be an STN problem, but rather a bad input problem, let's change the numbers in the input (ie. reverse the broken edge) to make this test pass.

File: src/stn.rs [hunk 2]
```diff
@@ -658,74 +658,545 @@ mod tests {
...
-  }
 
-  #[wasm_bindgen_test]
-  fn test_register_graph_converts_json_two_nodes_two_edges() {
-    let input = json!(
-      {
-        "edges": [{"minutes": 60, "source": 0, "target": 1}],
-      }
-    );
+    #[wasm_bindgen_test]
```
Comment (cameronwp) [inline src/stn.rs]: let's fix the indentation here

Comment (lanseafood) [reply src/stn.rs]: done

File: src/stn.rs (hunk 1 above)
Comment (lanseafood) [reply src/stn.rs]: changed minutes to 10 and reversed nodes

File: src/stn.rs [hunk 3]
```diff
@@ -719,14 +980,223 @@ mod tests {
...
+      Ok(u) => assert_eq!(
+          (4_usize, 12_usize),
+          (u.0, u.1),
+          "4 nodes, 12 edges expected to be made from given 3 edges"
+      ),
+      Err(e) => panic!("failed running stn.register_graph | {:?}", e),
+    }
+  }
+
+  // THIS TEST STILL FAILING
```
Comment (cameronwp) [inline src/stn.rs]: doesn't look like it's failing anymore :)
